modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -77,7 +77,7 @@
                                 "from inputStream\n" +
                                 "select iijtimestamp,value\n" +
                                 "insert into tempStream;" +
-                                "from tempStream#throughput:throughput(iijtimestamp,value)\n" +
+                                "from tempStream#throughput:throughput(iijtimestamp,\"latency\",1)\n" +
```

Comment:
Lets add @Parameter description for all parameters introduced

Reply:
Done